closure/goog/graphics/vmlelement.js

```diff
@@ -109,11 +109,16 @@ goog.graphics.VmlGroupElement.prototype.setSize = function(width, height) {
   var element = this.getElement();
 
   var style = element.style;
-  style.width = goog.graphics.VmlGraphics.toSizePx(width);
-  style.height = goog.graphics.VmlGraphics.toSizePx(height);
-
-  element.coordsize = goog.graphics.VmlGraphics.toSizeCoord(width) + ' ' +
+  style.width = /** @suppress {missingRequire} */
```

Comment:
@suppress {missingRequire} does not work in the middle of an expression.  At the very narrowest it should be on the statement, but having it on the function is also fine, particularly since it can be more clearly documented there.